HTTPStatusPlugin.php

```diff
@@ -20,12 +20,12 @@ class HTTPStatusPlugin {
   /**
    * Constants.
    */
-  const TABLENAME = 'jelle_s_http_status_codes';
-  const HTTP_STATUS_KEY = 'jelle_s_http_status_code';
-  const TABLE_ROW_FILTER = 'jelle_s_http_status_code_table_row';
-  const STATUS_CODE_OPTIONS_FILTER = 'jelle_s_http_status_code_options';
-  const SHUNT_SAVE_CODE = 'jelle_s_http_status_shunt_save_code';
-  const PRE_SAVE_CODE = 'jelle_s_http_status_pre_save_code';
```

Comment:
Won't this break existing installations?

Reply:
Hmmm, that is a good point, though it does that would only affect old installs since without the other updates it wouldn't have worked on any 1.7.3+ installs.

Actually as I think of it, all of this PRs changes will break for anyone already using the plugin, since the new database calls won't work with <1.7.3. I guess it would be good to just flag this as a breaking change either way, or else list it as a separate plugin for newer YourLS installs. I dunno, what do you think?

Reply:
Can't we do the update without the table renames?